Makefile

```diff
@@ -125,23 +128,28 @@ start-integration: web/src/build/static/app.js bin/goalert bin/psql-lite bin/wai
 jest: node_modules 
 	yarn workspace goalert-web run jest $(JEST_ARGS)
 
-test: node_modules jest
+test: node_modules jest ## Run all unit tests
 	go test -short ./...
 
 force-yarn:
 	yarn install --no-progress --silent --frozen-lockfile --check-files
 
-check: force-yarn generate node_modules
-	# go run ./devtools/ordermigrations -check
-	go vet ./...
-	go run github.com/gordonklaus/ineffassign ./...
-	CGO_ENABLED=0 go run honnef.co/go/tools/cmd/staticcheck ./...
+check: check-go check-js  ## Run all lint checks
+
+check-js: force-yarn generate node_modules
 	yarn run fmt
 	yarn run lint
 	yarn workspaces run check
 	./devtools/ci/tasks/scripts/codecheck.sh
 
-check-all: check test smoketest cy-wide-prod-run cy-mobile-prod-run
+check-go:
+	go vet ./...
+	go fmt ./...
```

Comment:
This won't cause a failure, only list unformatted files. It's already handled in `codecheck.sh` (which isn't go specific)
```suggestion
```

Reply:
Actually, it not only will list unformatted files, but it will also auto-correct them.  It'll fail if it can't auto correct it.

Reply:
We already run `gofmt` as part of `codecheck.sh` and fail if there are non-formatted (_or non-simplified_) files, so it's redundant. Auto-correcting isn't good enough for CI where we want it to fail if the committed code isn't formatted.

That said, we can move that check here so it's not redundant, I just don't want there to be two different places doing the same thing in two slightly different ways, we should pick one.

Reply:
If you run `go fmt ./...` on the master branch there's formatting errors, yet `make check` passes.  Based on the current behavior I would still say that `go fmt ./...` should be in there, as nothing else currently catches the formatting issues.

Reply:
removed gofmt as it doesn't actually do anything at this point.  Using `go fmt ./...` is more useful based on current behavior.